packages/web/components/LandingPage/Platform/styles.js

```diff
@@ -0,0 +1,57 @@
+import styled, { css } from 'styled-components';
+
+export const Container = styled.section`
+	${({ theme: { screens } }) => css`
+		display: flex;
+		align-items: center;
+		width: ${`${screens.large}px`};
+		max-width: 100%;
+		margin: 0 auto;
+		padding: 8rem 2.2rem;
```

Comment:
I think we might decrease this space on small or medium screens, cause it's too much...
<img width="377" alt="Screen Shot 2020-12-15 at 21 48 55" src="https://user-images.githubusercontent.com/12154623/102290230-58209b00-3f1f-11eb-9bcd-6a269b16b615.png">

Reply:
@lcnogueira This spacing is on the mobile design but I've kept the `8rem` spacing from desktop sizes. I agree that it's too spaced.